tests/test_torchscript.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Unit tests for exporting models via TorchScript.
+"""
+
+import regex
+import unittest
+
+import parlai.utils.testing as testing_utils
+from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
+from parlai.core.worlds import create_task
+from parlai.scripts.display_data import setup_args
+from parlai.torchscript.export_model import ScriptableGpt2BpeHelper
+from parlai.utils.bpe import Gpt2BpeHelper
+
+
+class TestTorchscript(unittest.TestCase):
+    def test_token_splitter(self):
+        """
+        Test TorchScriptable code for splitting tokens against reference GPT-2 version.
+        """
+
+        # Params
+        tasks = ['taskmaster2', 'convai2']
+        compiled_pattern = regex.compile(Gpt2BpeHelper.PATTERN)
+
+        with testing_utils.tempdir() as tmpdir:
+            datapath = tmpdir
+
+            for task in tasks:
+                parser = setup_args()
+                args = f"""\
```

Comment:
I would prefer if we use the standard ParlaiScript methodology so we could do the more parlai-onic:

`JitExport.parse_kwargs(task=task, datatype='train:ordered', datapath=tmpdir)`

Reply:
Yes, changed